main.go

```diff
@@ -252,6 +259,14 @@ func (p *prometheus) Serve() {
 		}
 	}()
 
+	hup := make(chan os.Signal)
+	signal.Notify(hup, syscall.SIGHUP)
+	go func() {
+		for range hup {
+			p.reloadConfig()
+		}
+	}()
+
 	notifier := make(chan os.Signal)
```

Comment:
If the above is called `hup` now, can we call this `term` or something like that?